framework/aster-frame/src/task/task.rs

```diff
@@ -278,52 +281,22 @@ impl TaskOptions {
 
         result.task_inner.lock().task_status = TaskStatus::Runnable;
         result.task_inner.lock().ctx.rip = kernel_task_entry as usize;
-        // Subtract 8 bytes to reserve space for the return address, otherwise
-        // we will write across the page bondary.
+        // We should reserve space for the return address in the stack, otherwise
+        // we will write across the page bondary due to the implementation of
+        // the context switch.
+        // According to system V AMD64 ABI, the stack pointer should be aligned
+        // to at least 16 bytes. And an alignment of 64 bytes is needed if a SIMD
```

Comment:
For an atomic type `T`, the natural alignment of `T` is the size of `T`. Passing 256-bit and 512-bit arguments does not necessarily means using SIMD instructions. So I think the comment should mention the size of arguments, not SIMD.

Reply:
Viewed the [SysV64 ABI](https://gitlab.com/x86-psABIs/x86-64-ABI) again, I think what you said is true.

Sorry for that.